pubsub/gcloud/aio/pubsub/subscriber_message.py

```diff
@@ -18,12 +18,14 @@ class SubscriberMessage:
     def __init__(self, ack_id: str, message_id: str,
                  publish_time: 'datetime.datetime',
                  data: Optional[bytes],
-                 attributes: Optional[Dict[str, Any]]):
+                 attributes: Optional[Dict[str, Any]],
+                 delivery_attempt: Optional[int]):
```

Comment:
Hmm just thought of something. I think this is technically a breaking change, since if anyone is instantiating `SubscriberMessage` directly now without specifying `delivery_attempt`, it will fail. Should be perhaps default this to `None` to avoid bumping major versions? @amartsinovich-dialpad @TheKevJames

Reply:
Ah, yes, good catch -- make this default to `None` for now, and if we think it _should_ be something users have to specify, we can add a TODO to deprecate this in a future release

Reply:
True - I'm not sure it is something users need to specify, as if they are not using deadlettering, it is of no concern to them.  I'll leave out the TODO.